Sources/SwiftDocC/Semantics/Reference/TabNavigator.swift

```diff
@@ -61,6 +61,11 @@ public final class TabNavigator: Semantic, AutomaticDirectiveConvertible, Markup
 extension TabNavigator {
     /// A container directive that holds general markup content describing an individual
     /// tab within a tab-based layout.
+    ///
+    ///  To add a new tab to a ``TabNavigator``, add  a `@Tab` directive within the content of the `@TabNavigator` directive.
+    ///
+    /// ## Topics
+    /// - ``TabNavigator``
```

Comment:
I _think_ this will create infinite recursion in the sidebar... Since TabNavigator owns Tab.
```suggestion
```